Classes/Issues/Comments/IssueCommentSectionController.swift

```diff
@@ -297,7 +297,9 @@ EditCommentViewControllerDelegate {
     // MARK: IssueCommentDetailCellDelegate
 
     func didTapMore(cell: IssueCommentDetailCell, sender: UIView) {
-        let alert = UIAlertController.configured(preferredStyle: .actionSheet)
+        guard let login = object?.details.login else { return }
```

Comment:
Do we want to fail completely (and silently) if we don't have a `login`?

Reply:
`didTapProfile` also has the same bahaviour!

Reply:
I'd still say let's just a quick toast error in before the return! 😅 I'd rather have _some_ indication of failure

Reply:
Added Generic error toast

Reply:
I wonder if we could make a `guardElseToast` function 😝

Reply:
We can at least just `return ToastManager.showGenericError()` inside the else?